cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -219,6 +219,20 @@
     } )
 }
 
+
+/* topdoc
+  name: Bulleted list with flush left alignment
+  family: cf-layout
+  notes:
+    - "Add flush left styling for bulleted lists"
+  tags:
+    - cf-layout
+*/
+
+.list__flush-left {
+    padding-left: 1em;
```

Comment:
Yep, as @Scotchester noted, this was trying to accomplish the same thing as the new standard. Fixed in https://github.com/cfpb/cfgov-refresh/pull/2876/commits/6542b7aa5674fb8ce8cc5b2787fb18b9d4ac5262.